packages/jupyter-widgets/src/manager/widget-manager.ts

```diff
@@ -16,6 +16,7 @@ import {
 } from "@nteract/core";
 import { JupyterMessage } from "@nteract/messaging";
 import { ManagerActions } from "../manager/index";
+import { initRequireDeps, overrideCDNBaseURL, requireLoader } from "./widgetLoader";
```

Comment:
I'm wondering if this could be 
```ts
import * as widgetLoader from "./widgetLoader"
```
instead of the individual functions, since it will be a bit clearer if you invoke them as widgetLoader.initRequireDeps, widgetLoader.overrideCDNBaseURL and widgetLoader.requireLoader

Reply:
Sure. Have updated the code to reflect this.